### PR TITLE
luci-app-statistics: update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by sotux**
fix some translation mistakes

### CODE REVIEW

File: applications/luci-app-statistics/po/zh-cn/statistics.po [hunk 1]
```diff
@@ -336,15 +336,15 @@ msgstr "忽略源地址"
 
 #: applications/luci-app-statistics/luasrc/model/cbi/luci_statistics/iptables.lua:103
 msgid "Incoming interface"
-msgstr "入接口"
+msgstr "传入接口"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/statistics.po]: The original translation is okay, maybe even more concise.  The same goes for "Outgoing interface"

File: applications/luci-app-statistics/po/zh-cn/statistics.po [hunk 2]
```diff
@@ -813,7 +813,7 @@ msgstr ""
 #: applications/luci-app-statistics/luasrc/model/cbi/luci_statistics/interface.lua:9
 msgid ""
 "The interface plugin collects traffic statistics on selected interfaces."
-msgstr "Interface 插件用于收集选定接口的数据包的统计信息。"
+msgstr "接口插件用于收集选定接口的数据包的统计信息。"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/statistics.po]: `s/数据包的/流量/`

File: applications/luci-app-statistics/po/zh-cn/statistics.po [hunk 3]
```diff
@@ -837,17 +837,17 @@ msgstr "iwinfo 插件收集无线信号强度、噪声和质量的统计信息
 
 #: applications/luci-app-statistics/luasrc/model/cbi/luci_statistics/load.lua:6
 msgid "The load plugin collects statistics about the general system load."
-msgstr "load 插件收集常规系统加载统计信息。"
+msgstr "负载插件收集常规系统负载的统计信息。"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/statistics.po]: 常规系统负载的统计信息 -> 系统负载的常规统计信息，或者省略掉general的翻译，几乎没有意义损失，同时更简练

File: applications/luci-app-statistics/po/zh-cn/statistics.po [hunk 4]
```diff
@@ -837,17 +837,17 @@ msgstr "iwinfo 插件收集无线信号强度、噪声和质量的统计信息
...
 msgid "The memory plugin collects statistics about the memory usage."
-msgstr "memory 插件收集关于内存使用情况的统计信息。"
+msgstr "内存插件收集关于内存使用情况的统计信息。"
 
 #: applications/luci-app-statistics/luasrc/model/cbi/luci_statistics/netlink.lua:11
 msgid ""
 "The netlink plugin collects extended information like qdisc-, class- and "
 "filter-statistics for selected interfaces."
-msgstr "netlink 插件收集为选定接口 qdisc-、class- 和 filter- 的扩展数据。"
+msgstr "netlink 插件为选定接口收集如 qdisc-、class- 和 filter- 的扩展数据。"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/statistics.po]: extended也可以不翻译。dash字符应当去掉，那个是英文原文中才需要使用的

File: applications/luci-app-statistics/po/zh-cn/statistics.po [hunk 5]
```diff
@@ -870,7 +870,7 @@ msgstr "ping 插件将发送 icmp echo replies 到选定主机来测量每台主
 msgid ""
 "The processes plugin collects information like cpu time, page faults and "
 "memory usage of selected processes."
-msgstr "processes 插件收集选定进程的 cpu 时间、页面错误和内存使用信息。"
+msgstr "处理器插件收集选定进程的 cpu 时间、页面错误和内存使用信息。"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/statistics.po]: processes应翻译为“进程”

**Comment by sotux** [comment]
@yousong , thanks. I've updated the traslation.

**Comment by yousong** [comment]
Thanks!

**Pull request merged**